databricks-example/scripts/split_intallments_payments.py

```diff
@@ -0,0 +1,9 @@
+import pandas
```

Comment:
I've seen insta`l`ments and insta`ll`ments (like the filename of this script) - Is it possible to use only one version?

Reply:
Would be using the single `l` spelling since this is used as the column name in kaggle (although the filename uses double `l`.